javaharness/java/arcs/android/service/ArcsService.java

```diff
@@ -43,7 +42,8 @@
   @Inject Arcs arcs;
   @Inject HarnessController harnessController;
   @Inject ShellApiBasedArcsEnvironment shellEnvironment;
-  @Inject PecPortManager pecPortManager;
+  @Inject
```

Comment:
nit - is the newline intentional? then it should probably be consistent across all data members.
also general question - is it better to declare data members like this, or have a constructor with all the injected objects (or it doesn't matter)?